Mage/src/main/java/mage/abilities/AbilitiesImpl.java

```diff
@@ -102,7 +102,7 @@ public AbilitiesImpl(final AbilitiesImpl<T> abilities) {
                     rules.add(sbRule.toString());
                 }
                 String rule = ability.getRule();
-                if (rule.length() > 0) {
```

Comment:
Good point.. Surely there must be a better way than to have Null Pointer Exceptions thrown from this though. Can we add server logging here and still do something with the rule text to default to - better than a NPE I'd hope?